proxy/src/CustomExpressServer.ts

```diff
@@ -54,34 +60,49 @@ export class CustomExpressServer {
 
     this._app.set("port", port);
     return new Promise<HttpServer>((resolve) => {
-      const server: HttpServer = this._app.listen(this._app.get("port"), () => resolve(server));
+      const server: HttpServer = this._app.listen(this._app.get("port"), () =>
+        resolve(server)
+      );
     });
   }
 
   // Creates a new Authorized request context with the passed in correlation id
   private async createContext(corrId: string | string[] | undefined) {
     let currCorrelationId = corrId;
-    if (undefined === currCorrelationId)
+    if (!currCorrelationId) {
       currCorrelationId = Guid.createValue();
-    else if (Array.isArray(currCorrelationId))
+    } else if (Array.isArray(currCorrelationId)) {
       currCorrelationId = currCorrelationId[0];
-
+    }
     const accessToken = await this._client.getAccessToken();
-    const ctx = new AuthorizedClientRequestContext(accessToken, currCorrelationId);
+    const ctx = new AuthorizedClientRequestContext(
+      accessToken,
+      currCorrelationId
+    );
     ctx.enter();
     return ctx;
   }
 
-  private async getNewUrl(ctx: AuthorizedClientRequestContext): Promise<string> {
-    if (undefined !== this._itwinUrl)
+  private async getNewUrl(
+    ctx: AuthorizedClientRequestContext
+  ): Promise<string> {
+    if (this._itwinUrl) {
       return this._itwinUrl;
+    }
     const urlClient = new UrlDiscoveryClient();
-    this._itwinUrl = await urlClient.discoverUrl(ctx, "iModelJsOrchestrator.K8S", undefined);
+    this._itwinUrl = await urlClient.discoverUrl(
```

Comment:
We should probably just take out the use of UrlDiscoveryClient. It's going to be deprecated soon.  The URL for the Visualization service is `https://api.bentley.com/imodeljs` at the moment so lets use that.